UnitsNet/Scripts/UnitDefinitions/ForcePerLength.json

```diff
@@ -0,0 +1,35 @@
+﻿{
+  "Name": "ForcePerLength",
+  "BaseUnit": "NewtonPerMeter",
+  "XmlDoc": "Force per length is the amount of change applied per length.",
+  "Units": [{
+      "SingularName": "NewtonPerMeter",
+      "PluralName": "NewtonsPerMeter",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Prefixes": [ "Nano", "Micro", "Milli", "Centi", "Deci", "Kilo" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "N/m" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "KilogramForcePerMeter",
```

Comment:
That sounds like a though nut to crack
And it can also prove important in #211 (since the unit is actually inversed and the PerUnit or PerMilliUnit might be an issue)